src/components/IndexTable/components/Row/Row.tsx

```diff
@@ -107,9 +107,10 @@ export const Row = memo(function Row({
         new MouseEvent(event.type, event.nativeEvent),
       );
     } else {
-      isNavigating.current = false;
```

Comment:
@lhoffbeck iirc you made a change *some what* recently regarding `isNavigating`. Does this change still work with your use-case?

Reply:
LGTM 👍. My change was to enable selecting by row when there isn't a `primaryLinkElement`, verified that @Brydom's change doesn't negatively affect the behavior.

Thanks for pinging @AndrewMusgrave!